deps.toml

```diff
@@ -13,7 +13,7 @@ junit-bom = "5.10.1"
 kotlin = "1.9.23"
 log4j = "2.21.1"
 lombok = "1.18.30"
-postgresql = "42.6.2"
+postgresql = "42.7.3"
```

Comment:
checking my understanding: this is what actually affects runtime code? i.e. the build.gradle only bumps testImplementation b/c there's some other `implementation` declaration somewhere, which references this version?